app/assets/javascripts/users.js

```diff
@@ -0,0 +1,25 @@
+function getRepositories() {
+  var url = $("#repositories").data("repositories")
+  $.getJSON(url, function(data) {
+    $.each(data, function(_idx, repository) {
+      $("#repositories").append("<tr>")
+      $("#repositories").append("<td>" + repository.name + "</td>")
+      $("#repositories").append("<td><a href=\"" + repository.html_url + "\">Go</a></td>")
+      $("#repositories").append("<td>" + repository.stargazers_count + "</td>")
+      $("#repositories").append("<td>" + repository.forks + "</td>")
+      $("#repositories").append("</tr>")
+    })
+  })
+}
+
+function getTemperature() {
+  var url = $("#temperature").data("temp")
+  $.getJSON(url, function(data) {
+    $("#temperature").append(data.temperature)
```

Comment:
Is this HTML safe? do you have a ´response_to´ in your controller? Are you appending strings to a DOM elemnt?